packages/apps/reputation-oracle/server/src/database/migrations/1719313669358-AddQualificationTable.ts

```diff
@@ -0,0 +1,99 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddQualificationTable1719313669358 implements MigrationInterface {
+  name = 'AddQualificationTable1719313669358';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "hmt"."site_keys" DROP CONSTRAINT "FK_178ba06ffb4808dbb40e782231e"`,
```

Comment:
This FK is already dropped here:
https://github.com/humanprotocol/human-protocol/blob/af4b1c187f9b3194e7b2c94878d0455746b6e531/packages/apps/reputation-oracle/server/src/database/migrations/1718970966064-RenameAndModifyUserRoleEnum.ts#L10